data/human/names.txt

```diff
@@ -3318,8 +3334,11 @@ phrase "intellectuals"
 		"Fourier"
 		"Franklin"
 		"Fresnel"
+		"Galilei"
```

Comment:
Isn't it:
```suggestion
		"Galileo"
```

Reply:
Galileo's last name is Galilei, hmm... but still could get changed.

Reply:
I don't think he's really known or thought of as Gelilei.